Longest_Increasing_subsequence_java/longest_increasing_subsequence_java.java

```diff
@@ -0,0 +1,48 @@
+package com.company;
+
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+
+    static int lengthOfLIS(int[] a) {
+        // checking if the size of the array is equal to 0 if it is then returning 0
+        if (a.length == 0) {
+            return 0;
+        }
+
+        int[] cache = new int[a.length];
+        Arrays.fill(cache, 1);
+
+        for (int i = 1; i < a.length; i++) {
+            for (int j = 0; j < i; j++) {
+                if (a[i] > a[j]) {
+                    cache[i] = Math.max(cache[i], cache[j] + 1); // checking for the longest increasing subsequence
```

Comment:
Move the comment to previous or next line. One line can't contain more than 80 characters including indentation.